gcc4mbed/Dockerfile

```diff
@@ -0,0 +1,45 @@
+#
+# To build, from base directory:
+#  docker build --tag shimaore/smoothie2-gcc4mbed gcc4mbed
+#
+FROM debian:testing
+MAINTAINER stephane@shimaore.net
+
+RUN \
+  dpkg --add-architecture i386 && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends \
+    ca-certificates \
+    lbzip2 \
+    make \
+    wget \
+    libc6:i386 \
+    libncurses5:i386 \
+    libstdc++6:i386
+
+COPY . /opt/smoothie/gcc4mbed
+WORKDIR /opt/smoothie/gcc4mbed
+
+RUN \
+  yes | ./linux_install && \
+  rm -r \
+    external/mbed/Release \
+    external/mbed/libraries/mbed/targets/cmsis/TARGET_ARM_SSG \
+    external/mbed/libraries/mbed/targets/cmsis/TARGET_Atmel \
+    external/mbed/libraries/mbed/targets/cmsis/TARGET_Freescale \
```

Comment:
Granted this is kind of ad-hoc — I was looking for large subtrees I could remove without breaking the Smoothie2 build, and as I understand things we only need `TARGET_NXP` for Smoothie2. But it does make the first container much smaller.

(Hmmm apparently I left my original attempts at making the build smaller in the `gcc4mbed/.dockerignore` file -- the build was failing with those. I guess I could clean-up the dockerignore file.)